src/run_perf_test.py

```diff
@@ -45,13 +49,19 @@ def main():
     manifest = BundleManifest.from_file(args.bundle_manifest)
     config = yaml.safe_load(args.config)
 
+    tests_dir = os.path.join(os.getcwd(), "test-results", "perf-test")
+    os.makedirs(tests_dir, exist_ok=True)
+
     with TemporaryDirectory(keep=args.keep, chdir=True) as work_dir:
         current_workspace = os.path.join(work_dir.name, "infra")
         with GitRepository(get_infra_repo_url(), "main", current_workspace):
-            security = "security" in manifest.components
```

Comment:
Shouldn't we get security from bundle manifest?

Reply:
That would prevent us from parallelizing the tests in the future. We want to kick off two tests for the security based bundles, which can be done from the Jenkins pipeline

Reply:
I feel like security enabled/disabled should be decided based on `security` param in `manifest.components`. To parallelize the test we can pass 2 different bundle manifest, one with security enabled and other disabled.

Reply:
We will have to publish these manifests to our buckets and distributions, which we do not currently.
Moving it out additionally gives more flexibility for extensibility moving forward for other use cases rather than updating/adding manifests for every build.

Reply:
We do publish manifests to our buckets. `without-security` tests is something that will run irrespective of what's in the manifest but for `with-security` can we add a check in the codebase and then start the test only if the security component is present?
If the performance tests are to be run nightly it is highly possible that security component is not present initially in the manifest.

Reply:
Refer to the comment below - https://github.com/opensearch-project/opensearch-build/pull/1671#discussion_r828477283

What I was trying to say is - we have a single manifest which might or might not have security in the components. By enabling the logic at the Jenkins level, we can execute the two runs in parallel, giving us more control over the test runs and statuses. 

Also, it is based directly on the manifest. Look here for more - https://github.com/opensearch-project/opensearch-build/pull/1671/files#diff-4debf5e3ece07145d8395f15df88f49b8b784a274cead94e2a94c8b7152c11efR75

All I am doing is pulling out that logic for better control at the top level of job execution.